build.zig

```diff
@@ -40,7 +40,7 @@ pub fn build(b: *std.build.Builder) void {
         .{ .name = "instanced-cube", .packages = &[_]Pkg{Packages.zmath} },
         .{ .name = "advanced-gen-texture-light", .packages = &[_]Pkg{Packages.zmath} },
         .{ .name = "fractal-cube", .packages = &[_]Pkg{Packages.zmath} },
-        .{ .name = "gkurve", .packages = &[_]Pkg{ Packages.zmath, Packages.zigimg, freetype.pkg }, .std_platform_only = true },
+        .{ .name = "gkurve", .packages = &[_]Pkg{ Packages.zmath, Packages.zigimg, freetype.freetype_pkg }, .std_platform_only = true },
```

Comment:
I wonder if `freetype.pkg` and `freetype.harfbuzz_pkg` are not better names? `freetype.freetype_pkg` seems a little repetitive.

In any case, we should update README example for how to use.

Reply:
right, will fix this in next PRs